### PR TITLE
v1.0.x

### PR DESCRIPTION
**Pull request by jasonmadigan**
- v1.0.0 updates

branch is a cul-dec-sac for 1.x, should not be merged (just created a PR for builds and previews)

### CODE REVIEW

File: docs/getting-started-single-cluster.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ If you want to make use of the Kuadrant `DNSPolicy` you should setup the followi
 ### Set the release you want to use 
 
 ```bash
-export KUADRANT_REF=main
+export KUADRANT_REF=v1.0.0
```
Comment (maleck13) [inline docs/getting-started-single-cluster.md:54]: I would prob just set this to ```release-v1.0.0``` that way we can fix minor issues with the scripts etc

Comment (jasonmadigan) [reply docs/getting-started-single-cluster.md:54]: makes sense

**Comment by jasonmadigan** [comment]
Some broken links:

```
WARNING -  Doc file 'authorino/docs/user-guides/authenticated-rate-limiting-envoy-dynamic-metadata.md' contains a link './features.md#common-feature-common-expression-language-cel', but the target 'authorino/docs/user-guides/features.md' is not found among documentation files.
WARNING -  Doc file 'authorino/docs/user-guides/deny-with-redirect-to-login.md' contains a link './features.md#common-feature-common-expression-language-cel', but the target 'authorino/docs/user-guides/features.md' is not found among documentation files.
WARNING -  Doc file 'authorino/docs/user-guides/external-metadata.md' contains a link './features.md#common-feature-common-expression-language-cel', but the target 'authorino/docs/user-guides/features.md' is not found among documentation files.
WARNING -  Doc file 'authorino/docs/user-guides/http-basic-authentication.md' contains a link './features.md#common-feature-common-expression-language-cel', but the target 'authorino/docs/user-guides/features.md' is not found among documentation files.
WARNING -  Doc file 'authorino/docs/user-guides/injecting-data.md' contains a link './features.md#common-feature-common-expression-language-cel', but the target 'authorino/docs/user-guides/features.md' is not found among documentation files.
WARNING -  Doc file 'authorino/docs/user-guides/json-pattern-matching-authorization.md' contains a link './features.md#common-feature-common-expression-language-cel', but the target 'authorino/docs/user-guides/features.md' is not found among documentation files.
INFO    -  Doc file 'kuadrant-operator/doc/auth.md' contains a link 'rate-limiting.md#limitation-multiple-network-resources-with-identical-hostnames', but the doc 'kuadrant-operator/doc/rate-limiting.md' does not contain an anchor '#limitation-multiple-network-resources-with-identical-hostnames'.
```

**Comment by jasonmadigan** [comment]
Waiting for https://github.com/Kuadrant/kuadrant-operator/pull/1046

**Comment by jasonmadigan** [comment]
/hold

branch is a cul-dec-sac for 1.x, should not be merged (just created a PR for builds and previews)

**Pull request closed without merging**